benchmark/misc/util-extend-vs-object-assign.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const common = require('../common.js');
+const util = require('util');
+const v8 = require('v8');
+
+const bench = common.createBenchmark(main, {
+  type: ['util._extend', 'Object.assign',
+         'util._extend', 'Object.assign'],
+  n: [10e4]
+});
+
+function main(params) {
+  let fn;
+  const n = params.n | 0;
+  let v8command;
+
+  if (params.type == 'util._extend') {
+    fn = util._extend;
+    v8command = '%OptimizeFunctionOnNextCall(util._extend)';
```

Comment:
I don't know exactly how `OptimizeFunctionOnNextCall()` works/is implemented, but I wonder if `util._extend` here should instead be `fn`?